snippets/partition_options.mdx

```diff
@@ -0,0 +1,25 @@
+- `table_extraction_options`: A map with string keys specifying options for table extraction, which currently only supports the boolean `include_additional_text`, which will add in text from OCR boxes.  When include_additional_text is set to true and table extraction is enabled, DocParse will attempt to enhance the table structure by merging in tokens from text extraction. This can be useful for working with tables that have missing or misaligned text. `include_additional_text` is `False` by default. The default table_extraction_options is `{}`.
+- `extract_images`: A boolean that determines whether to extract images from the document. Default: `False`.
+- `ocr_images`: A boolean that, when `True`, causes DocParse to use OCR to attempt to generate a text representation of detected images. When `False`, images do not contain a text_representation. Default is `False`.
+- `selected_pages`: A list specifying individual pages (1-indexed) and page ranges from the document to partition. Single pages are specified as integers and ranges are specified as lists with two integer entries in ascending order. A valid example value for selected_pages is `[1, 10, [15, 20]]` which would include pages 1, 10, 15, 16, 17 ..., 20.  `selected_pages` is `None` by default, which results in all pages of the document being parsed.
+- `chunking_options`: A dictionary of options for specifying chunking behavior. Chunking is only performed when this option is present, and default options are chosen when `chunking_options` is specified as `{}`.
+    - `strategy`: A string specifying the strategy to use to combine and split chunks. Valid values are `context_rich`, `mixed_multi_column`, and `maximize_within_limit`. The default and recommended chunker is `context_rich` as `{'strategy': 'context_rich'}`.
+        - Behavior of `context_rich` chunker: The goal of this strategy is to add context to evenly-sized chunks. This is most useful for retrieval based GenAI applications. context_rich chunking combines adjacent `Section-header` and `Title` elements into a new `Section-header` element. Merges elements into a chunk with its most recent `Section-header`. If the chunk would contain too many tokens, then it starts a new chunk copying the Section-header to the start of this new chunk and continues. Merges elements on different pages, unless `merge_across_pages` is set to `False`.
+
+        - Behavior of `mixed_multi_column` chunker: `mixed_multi_column` performs well on journal articles and documents layouts that contain one or two columns. For each page, it detects whether the elements are present in two columns and if so, it orders them properly with the left column appearing in the order first. Then, it drops elements that are only one token in size. It also drops elements in the top and bottom 5% of the page. Elements returned do not have a `type`. The `marked` merger always breaks across pages.
```

Comment:
"documents layouts" wording is weird here. 

What does "performs well mean"?  

I don't think I understand what's going on here.

Reply:
I will change this to "document layouts"

Reply:
We can add a picture to explain this.

Reply:
Say "is meant for" instead of "performs well"